src/containers/shared/components/TokenSearchResults/styles.scss

```diff
@@ -0,0 +1,141 @@
+@import '../../css/variables';
+
+.search-results-menu {
```

Comment:
This file has a lot of magic numbers that should be moved to variables.

Reply:
I will create an issue to fix this later